client/components/Profile/ProfilePictureForm/ProfilePictureForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Field, reduxForm } from 'redux-form'
```

Comment:
We are trying to phase out `redux-form` (see [here](https://www.pivotaltracker.com/story/show/154235506))

The CreateProjectForm is an example of how @sexta13 tackled this (you may need to pull those changes as I just merged the PR)